pkg/resources/objects.go

```diff
@@ -34,3 +36,28 @@ func GetJsonPathString(obj map[string]interface{}, path string) (string, error)
 	}
 	return buf.String(), nil
 }
+
+// GetJsonPathValue extracts the value from the object using given JsonPath template
+func GetJsonPathValue(obj interface{}, path string) (interface{}, error) {
+	j := jsonpath.New("GetJsonPathValue")
+	// If the key is missing, return an empty string without errors
+	j.AllowMissingKeys(true)
+	err := j.Parse(path)
+	if err != nil {
+		return nil, fmt.Errorf("JsonPath parse %s error: %v", path, err)
+	}
+	values, err := j.FindResults(obj)
+	if err != nil {
+		return nil, fmt.Errorf("JsonPath execute error: %v", err)
+	}
+	if len(values) == 0 {
```

Comment:
Not sure if it will, I prefer to run preconditions check with return before the actual processing logic.